### PR TITLE
Adding a couple of notes to the quickstart document

### PR DESCRIPTION
**Pull request by wainersm**
I spent some days trying to deploy and use CoCo on AMD SEV. I learned some stuffs that I think would be useful to have recorded in the quickstart document. Also I found some outdated information that I fixed.

### CODE REVIEW

**Review by fidencio — COMMENTED** [review]
@wainersm, thanks for the contribution, it looks fine to me apart from the comments made as part of the review.

**Comment by fidencio** [comment]
I've updated @wainersm's branch.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @wainersm!

File: quickstart.md [hunk 1]
```diff
@@ -1061,6 +1070,23 @@ Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create
 Unfortunately this is a generic message. You'll need to go deeper to figure out
 what is going on.
 
+### failed to create shim task: failed to mount "/run/kata-containers/shared/containers/CONTAINER_NAME/rootfs"
```
Comment (fitzthum) [inline quickstart.md:1073]: Maybe we should move this under the VM does start section below?

**Review by fitzthum — APPROVED** [review]
Great additions.

**Pull request merged**